src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -97,7 +97,7 @@ public void removeEatery(Eatery key) {
 
     @Override
     public String toString() {
-        return eateries.asUnmodifiableObservableList().size() + " eateries";
+        return eateries.asUnmodifiableObservableList() + " eateries";
```

Comment:
Why has the size() been removed? wouldnt it impact what is returned

Reply:
uh it changes the error message only? also i feel this one more useful bc u can see what's in the list rather than just how many items in the list hahaha

Reply:
wouldnt there be a super long error message if the list is long

Reply:
ya but it's still more useful than "expected: 98 eateries actual: 98 eateries"

Reply:
hmm I think both will be good, the number for quick comparison, and the list to look for the difference